crates/cards/Cargo.toml

```diff
@@ -12,10 +12,13 @@ default = ["serde"]
 time = { version = "0.3.20" }
 error-stack = "0.3.1"
 serde = { version = "1", features = ["derive"], optional = true }
+serde_json = "1.0.96"
```

Comment:
Do we need `serde_json` to be among our dependencies, or would it suffice to have it in `dev-dependencies`?

Reply:
> Everything else looks good. Thanks for keeping up with the changes on the `main` branch. I can take it up from here, if you'd prefer that.

there are still some CI errors that I am not too sure about as I don't get these locally.

Reply:
I'll handle it then, no worries!

Reply:
thank you. and I believe `serde_json` is sufficient in the `dev-dependencies`